### ISSUE TITLE
Navigation with Tab key

### ISSUE THREAD

**Issue by Ident** [cegui/cegui#51]
**[Original report](https://bitbucket.org/cegui/cegui/issue/51) by Anonymous.**

----------------------------------------

**Automatic migration. Original reporter: "mrbolha"**

It should be possible to switch the component with the keyboard focus by pressing the "Tab" key. Also, "Shift-Tab" should be used to navigate backwards.

Optionally, components like "MultiLineEditbox" might want to handle "Tab" keystrokes, generating "extra spaces" in the text, and not changing the Focus.

**Reproducibility:** N/A
